flare_portal/api/tests/test_views.py

```diff
@@ -22,40 +23,41 @@
 test_file = "flare_portal/experiments/tests/assets/circle.png"
 
 
+def get_example_experiment() -> Experiment:
+    us_file = SimpleUploadedFile("file.wav", b"wav content", content_type="audio/wav")
+    csa_file = SimpleUploadedFile(
+        "csa.png", open(test_file, "rb").read(), content_type="image/png"
+    )
+    csb_file = SimpleUploadedFile(
+        "csa.png", open(test_file, "rb").read(), content_type="image/png"
+    )
+    context_a_file = SimpleUploadedFile(
+        "context_a.png",
+        open(test_file, "rb").read(),
+        content_type="image/png",
+    )
+    context_b_file = SimpleUploadedFile(
+        "context_b.png",
+        open(test_file, "rb").read(),
+        content_type="image/png",
+    )
+    return ExperimentFactory(
+        us=us_file,
+        csa=csa_file,
+        csb=csb_file,
+        context_a=context_a_file,
+        context_b=context_b_file,
+    )
+
+
```

Comment:
This is the sort of thing we should add as part of the factory, but happy to leave this is as is for now.